### PR TITLE
Post and Get from data from API

### PR DESCRIPTION
**Pull request by leonard33**
Using fetch to post and get data from  the game api

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,33 +1,50 @@
...
+  const scorestream = await fetch(url);
+  const scoredata = await scorestream.json();
+  scorelist.innerHTML = '';
+  // eslint-disable-next-line array-callback-return
+  scoredata.result.forEach((key) => {
+    scorelist.innerHTML += `<ul class="scoreput">
+    <li class="names">${key.user}:&nbsp;${key.score}</li>
+  </ul>`;
+  });
+}
```
Comment (EricMbouwe) [inline src/index.js]: - [ ] Let's use `arrow functions` as required for this project.

File: src/index.js [hunk 2]
```diff
@@ -1,33 +1,50 @@
...
+      user: username,
+      score: userscore,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  })
+    .then((response) => response.json())
+    .then((json) => json);
+}
```
Comment (EricMbouwe) [inline src/index.js]: - [ ] Let's use `arrow functions` as required for this project.

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @leonard33, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Working app

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- NA

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by leonard33** [comment]
PR correction:
- Use of ES6 arrow functions as per the project review

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @leonard33, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**